docs/platform-validation/index.md

```diff
@@ -0,0 +1,141 @@
+
+This is also meaningful test case for the carbon footprint accuracy check on AI/AIGC workloads. 
+
+## Validation workflow
+
+See example Github action workflow [here](https://github.com/sustainable-computing-io/kepler/blob/main/.github/workflows/platform-validation.yml).
+
+The workflow is similar as the e2e integration test workflow, has two phases jobs:
+
+### Artifacts build.
+
+Build both the `Kepler` container image and the `Validator` container image.
+
+The current Validator image refers to the Kepler image build mechanism to build from a GPU image to support GPU case. We may simply build it from UBI image later.
+
+Since it is an independent energy collector other than Kepler, those eBPF stuffs will not be bumped into the container.
+
+### Platform validation test.
+
+The major steps in this job are as follows:
+
+1. Download artifacts and import images.
+
+2. Run `validator` image to collect energy and calculate node components power consumption data before deploying local-dev-cluster(Kind in this example)
+
+3. Use Kepler action to deploy local-dev-cluster(Kind) with local registry support and full kube-prometheus monitoring stack.
+
+4. Run `validator` image again to collect energy and calculate node components power consumption data before deploying Kepler exporter. 
+
+5. Deploy Kepler exporter in local-dev-cluster above and let it be monitored by Prometheus.
+
+6. Run `validator` image the 3rd time to collect energy and calculate node components power consumption data after deploying Kepler exporter.
+
+7. Run platform-validation test scripts, do below things:
+
+(1) `port forwarding` kepler-exporter and prometheus-k8s service to let their ports accessible to Ginkgo test code.
+
+(2) Use `ginkgo` CLI to run [test suite](https://github.com/sustainable-computing-io/kepler/blob/main/e2e/platform-validation), execute cases described in section [Mechanism and methodology](https://github.com/sustainable-computing-io/kepler-doc/blob/main/docs/platform-validation/index.md#mechanism-and-methodology) above and generate test report in `platform_validation_report.json` file assigned by `--json-report` parameter.
+
+(3) Dump necessary test log and data for post-test evaluation use.
+
+In specific test cases, the power data delta of step 4 and 2 could be comparison base for local-dev-cluster's power consumption; while the power data delta of step 6 and 4 could be the kepler exporter's power comparison base.
+
+8. Cleanup the test environment(undeploy Kepler exporter, the local-dev-cluster and the local registry).
+
+
+## Validation result evaluation.
+
+Check validation report for any failure cases.
+
+Check accuracy cases comparison result. 
+
+Node level cases should be more accurate than pod/container level cases, make analysis on any abnormal results and figure out RCAs for them.
```

Comment:
TODO create an analyzer for abnormality detection

Reply:
Yeah, this is really TODO item which I am doing, will update when it is ready.